server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -40,53 +39,6 @@ allExciRules: cics_exci_link | cics_exci_delete | cics_exci_delete_container | c
 
 allSPRules: cics_acquire_terminal | cics_disable | cics_discard | cics_enable | cics_extract_system_programming | cics_inquire_system_programming | cics_create | cics_perform | cics_resync_entryname | cics_collect_statistics | cics_csd;
 
-// compiler options
-compilerOpts
-   : (XOPTS LPARENCHAR compilerXOptsOption (commaClause? compilerXOptsOption)* RPARENCHAR) | cicsOptions;
-
-cicsOptions:  CICS LPARENCHAR (cicsTranslatorCompileDirectivedKeywords | NONNUMERICLITERAL)  RPARENCHAR;
-
-compilerXOptsOption
-    : APOST |
-      CBLCARD |
-      CICS |
-      COBOL2 |
-      COBOL3 |
-      CPSM |
-      DBCS |
-      DEBUG |
-      DLI |
-      EDF |
-      EXCI |
-      FEPI |
-      ((FLAG | F_CHAR) LPARENCHAR (E_CHAR | I_CHAR | S_CHAR | U_CHAR | W_CHAR) (commaClause (E_CHAR | I_CHAR | S_CHAR | U_CHAR | W_CHAR))? RPARENCHAR) |
-      LENGTH |
-      ((LINECOUNT | LC) LPARENCHAR integerLiteral RPARENCHAR) |
-      LINKAGE |
-      NATLANG |
-      NOCBLCARD |
-      NOCPSM |
-      NODEBUG |
-      NOEDF |
-      NOFEPI |
-      NOLENGTH |
-      NOLINKAGE |
-      NONUM |
-      NOOPTIONS |
-      NOSEQ |
-      NOSPIE |
-      NOVBREF |
-      NUM |
-      OPTIONS |
-      QUOTE |
-      SEQ |
-      SP |
-      SPACE LPARENCHAR integerLiteral RPARENCHAR |
-      SPIE |
-      SYSEIB |
-      VBREF
-    ;
-
 cicsTranslatorCompileDirectivedKeywords
```

Comment:
This rule could be now inlined into `cicsWords` and some of the tokens could be removed from the lexer.